openlibrary/templates/jsdef/LazyWorkPreview.html

```diff
@@ -21,10 +21,7 @@
                 $_('by') <span class="authors">${', '.join(work['author_name'])}</span>
         </span>
         &bull;
-        $if work['edition_count'] == 1:
-            <span class="edition_count">$work['edition_count'] edition</span>
-        $else:
-            <span class="edition_count">$work['edition_count'] editions</span>
+        <span class="edition_count">$ungettext('One edition', '%(count)d editions', work['edition_count'], count=work['edition_count'])</span>
```

Comment:
```suggestion
        <span class="edition_count">$ungettext('%(count)d edition', '%(count)d editions', work['edition_count'], count=work['edition_count'])</span>
```
This is on the theory the display could say `0 edition` or `0 editions`, which maybe it can't.

Reply:
I think that's the right call, thank you!